mlflow/keras.py

```diff
@@ -67,13 +67,10 @@ def get_default_conda_env(include_cloudpickle=False, keras_module=None):
     # The Keras pyfunc representation requires the TensorFlow
     # backend for Keras. Therefore, the conda environment must
     # include TensorFlow
-    if LooseVersion(tf.__version__) < LooseVersion('2.0.0'):
+    if LooseVersion(tf.__version__) <= LooseVersion('1.13.2'):
```

Comment:
is this because tf versions greater than `1.13.2` are not available on conda?

Reply:
Yup, we have the `keras`/`tf.keras` tests running for 1.X and 2.0 (2.0 was already running before). 

According to https://anaconda.org/conda-forge/tensorflow, the latest version available is 1.13.2 :/

Reply:
Alternatively, we could use `tensorflow-gpu`? https://anaconda.org/anaconda/tensorflow-gpu

Reply:
I don't think we need to change it - i was just curious